firefox-ios/firefox-ios-tests/Tests/ClientTests/Application/DefaultBrowserUitTests.swift

```diff
@@ -0,0 +1,103 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import Shared
+@testable import Client
+
+final class DefaultBrowserUitTests: XCTestCase {
```

Comment:
Nit: is this supposed to be `DefaultBrowserUtilTests` instead of `DefaultBrowserUitTests`?